[INFO]-[2019-09-06 16:51:49,735]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 23136 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 16:51:50,379]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 16:51:51,242]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 16:51:51,717]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$30470332] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 16:51:52,572]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 16:51:53,144]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 16:51:53,386]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 16:51:53,747]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 16:51:54,124]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 16:51:54,147]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 16:51:54,180]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 16:51:54,286]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 5.107 seconds (JVM running for 6.545)
[INFO]-[2019-09-06 16:51:54,483]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-f74643f4601a800b56cbc50fec8964cfb1fe378f
[INFO]-[2019-09-06 16:51:54,500]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 16:51:54,501]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:03:12,789]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 3536 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:03:13,411]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:03:14,109]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:03:14,443]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$c9322b6b] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:03:15,220]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:03:15,690]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:03:15,866]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:03:16,191]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:03:16,510]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:03:16,535]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:03:16,564]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:03:16,639]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 4.202 seconds (JVM running for 4.891)
[INFO]-[2019-09-06 17:03:16,812]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-f74643f4601a800b56cbc50fec8964cfb1fe378f
[INFO]-[2019-09-06 17:03:16,863]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:03:16,864]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:03:27,385]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 13060 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:03:28,090]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:03:28,871]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:03:29,217]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$21cf8a8c] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:03:30,068]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:03:30,544]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:03:30,710]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:03:31,056]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:03:31,385]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:03:31,410]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:03:31,439]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:03:31,519]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 4.594 seconds (JVM running for 5.425)
[INFO]-[2019-09-06 17:03:31,661]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-f74643f4601a800b56cbc50fec8964cfb1fe378f
[INFO]-[2019-09-06 17:03:31,703]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:03:31,703]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:28:29,378]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 19672 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:28:30,178]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:28:31,024]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:28:31,363]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$21cf8a8c] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:28:32,236]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:28:32,704]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:28:32,922]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[ERROR]-[2019-09-06 17:28:37,648]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:922]-init datasource error, url: jdbc:mysql://47.104.225.53:10005/wchat?autoReconnect=true&autoReconnectForPools=true&useUnicode=true&characterEncoding=utf8
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:905)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:830)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:918)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1903)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1846)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1774)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:593)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:515)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:849)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:877)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:127)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:117)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:108)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'117.158.134.217' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:816)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:741)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:709)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:132)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:540)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:202)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1449)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:165)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:849)
	... 56 common frames omitted
[ERROR]-[2019-09-06 17:28:37,961]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:966]-{dataSource-1} init error
java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:905)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:830)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:918)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1903)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1846)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1774)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:593)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:515)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:849)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:877)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:127)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:117)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:108)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'117.158.134.217' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:816)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:741)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:709)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:132)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:540)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:202)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1449)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:165)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:849)
	... 56 common frames omitted
[INFO]-[2019-09-06 17:28:37,963]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[WARN]-[2019-09-06 17:28:37,964]-[192.168.30.2:-1]-[]-[org.springframework.context.support.AbstractApplicationContext:557]-Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/alibaba/druid/spring/boot/autoconfigure/DruidDataSourceAutoConfigure.class]: Invocation of init method failed; nested exception is java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
[INFO]-[2019-09-06 17:28:37,964]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:28:37,976]-[192.168.30.2:-1]-[]-[org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener:142]-

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
[ERROR]-[2019-09-06 17:28:37,977]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:858]-Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/alibaba/druid/spring/boot/autoconfigure/DruidDataSourceAutoConfigure.class]: Invocation of init method failed; nested exception is java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:593)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:515)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:849)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:877)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:127)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:117)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:108)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:905)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:830)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:918)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1903)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1846)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1774)
	... 42 common frames omitted
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'117.158.134.217' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:816)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:741)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:709)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:132)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:540)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:202)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1449)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:165)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:849)
	... 56 common frames omitted
[ERROR]-[2019-09-06 17:28:37,979]-[192.168.30.2:-1]-[]-[org.springframework.test.context.TestContextManager:250]-Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@3270d194] to prepare test instance [com.liupeihao.wchat.WchatApplicationTests@1a538ed8]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:125)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:108)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [com/alibaba/druid/spring/boot/autoconfigure/DruidDataSourceAutoConfigure.class]: Invocation of init method failed; nested exception is java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1778)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:593)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:515)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:320)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:318)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:849)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:877)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:127)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:117)
	... 27 common frames omitted
Caused by: java.sql.SQLNonTransientConnectionException: Could not create connection to database server. Attempted reconnect 3 times. Giving up.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:110)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:905)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:830)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:455)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1578)
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)
	at com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:918)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1903)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1846)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1774)
	... 42 common frames omitted
Caused by: com.mysql.cj.exceptions.CJException: Access denied for user 'root'@'117.158.134.217' (using password: YES)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:129)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:816)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:741)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:709)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:132)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:540)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:202)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1449)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:165)
	at com.mysql.cj.jdbc.ConnectionImpl.connectWithRetries(ConnectionImpl.java:849)
	... 56 common frames omitted
[INFO]-[2019-09-06 17:28:56,550]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 20508 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:28:57,224]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:28:58,005]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:28:58,385]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$21cf8a8c] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:28:59,159]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:28:59,609]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:28:59,776]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:29:04,161]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:29:04,480]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:29:04,503]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:29:04,533]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:29:04,625]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.426 seconds (JVM running for 9.123)
[INFO]-[2019-09-06 17:29:04,782]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-f74643f4601a800b56cbc50fec8964cfb1fe378f
[INFO]-[2019-09-06 17:29:04,821]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:29:04,821]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:36:41,913]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 17784 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:36:42,567]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:36:43,373]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:36:43,734]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$21cf8a8c] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:36:44,534]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:36:45,021]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:36:45,212]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:36:49,813]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:36:50,116]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:36:50,136]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:36:50,158]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:36:50,238]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.687 seconds (JVM running for 9.429)
[INFO]-[2019-09-06 17:36:50,402]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-f74643f4601a800b56cbc50fec8964cfb1fe378f
[INFO]-[2019-09-06 17:36:50,453]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:36:50,454]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:40:36,345]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 24204 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:40:36,998]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:40:37,783]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:40:38,242]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$e2f08cba] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:40:39,125]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:40:39,689]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:40:39,922]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:40:44,453]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:40:44,810]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:40:44,836]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:40:44,870]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:40:44,980]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.143 seconds (JVM running for 10.665)
[INFO]-[2019-09-06 17:40:45,153]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-b38f14ff92e2db206c7618f065a8466a9b2c11b9
[INFO]-[2019-09-06 17:40:45,173]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:40:45,175]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:41:20,886]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 17456 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:41:21,542]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:41:22,337]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:41:22,813]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$54b55b86] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:41:23,715]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:41:24,272]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:41:24,481]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:41:29,072]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:41:29,430]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:41:29,456]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:41:29,490]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:41:29,619]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.246 seconds (JVM running for 10.727)
[INFO]-[2019-09-06 17:41:29,787]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-5adeaa806f0ca25e8b39dc2e24466d1cceda574c
[INFO]-[2019-09-06 17:41:29,803]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:41:29,804]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:43:22,214]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 5788 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:43:22,813]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:43:23,617]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:43:24,065]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$e2f08cba] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:43:24,942]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:43:25,543]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:43:25,811]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:43:30,139]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:43:30,521]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:43:30,549]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:43:30,586]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:43:30,681]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.969 seconds (JVM running for 10.836)
[INFO]-[2019-09-06 17:43:30,847]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-0acefb2bfc5a64dbee1c1ea4e400380d9f4c4162
[INFO]-[2019-09-06 17:43:30,862]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:43:30,863]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:44:28,814]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 22804 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:44:29,445]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:44:30,178]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:44:30,509]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$9d03444d] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:44:31,298]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:44:31,798]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:44:31,987]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:44:36,279]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:44:36,577]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:44:36,599]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:44:36,621]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:44:36,694]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.242 seconds (JVM running for 8.912)
[INFO]-[2019-09-06 17:44:36,842]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-0acefb2bfc5a64dbee1c1ea4e400380d9f4c4162
[INFO]-[2019-09-06 17:44:36,882]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:44:36,883]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:45:29,958]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 17468 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:45:30,605]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:45:31,323]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:45:31,684]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$c9322b6b] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:45:32,456]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:45:32,930]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:45:33,096]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:45:37,451]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:45:37,743]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:45:37,769]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:45:37,799]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:45:37,887]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.294 seconds (JVM running for 8.955)
[INFO]-[2019-09-06 17:45:38,039]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-0acefb2bfc5a64dbee1c1ea4e400380d9f4c4162
[INFO]-[2019-09-06 17:45:38,082]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:45:38,082]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:48:28,069]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 18960 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:48:28,778]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:48:29,589]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:48:30,050]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$33a06076] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:48:30,958]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:48:31,567]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:48:31,804]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:48:36,341]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:48:36,764]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:48:36,787]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:48:36,834]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:48:36,934]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.418 seconds (JVM running for 10.894)
[INFO]-[2019-09-06 17:48:37,114]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-4974c24f1c0c8cd3bb57f1d0e56d7d1576f483bf
[INFO]-[2019-09-06 17:48:37,128]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:48:37,129]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:52:42,193]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 20556 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 17:52:42,803]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 17:52:43,597]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 17:52:44,060]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$c6481088] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 17:52:44,934]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 17:52:45,542]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 17:52:45,785]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 17:52:50,294]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 17:52:50,683]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 17:52:50,709]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 17:52:50,742]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 17:52:50,843]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.149 seconds (JVM running for 10.59)
[INFO]-[2019-09-06 17:52:51,026]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:29]-e46f0bac7fa9d1ff575215d63cb218f6bee61041
[INFO]-[2019-09-06 17:52:51,042]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 17:52:51,043]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:08:10,721]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 24792 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 18:08:11,427]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 18:08:12,398]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 18:08:12,996]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$695c91c3] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 18:08:13,979]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 18:08:14,680]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:08:14,899]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 18:08:19,271]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 18:08:19,658]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 18:08:19,681]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 18:08:19,719]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 18:08:19,838]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.596 seconds (JVM running for 11.29)
[INFO]-[2019-09-06 18:08:19,993]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:31]-15677634721889789072liupeihao
[INFO]-[2019-09-06 18:08:19,994]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:33]-1b462e6d1528b7461a9bf544960fc322e0240b1f
[INFO]-[2019-09-06 18:08:20,014]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 18:08:20,015]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:09:32,942]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 22420 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 18:09:33,578]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 18:09:34,450]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 18:09:34,895]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$1269b2da] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 18:09:35,755]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 18:09:36,318]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:09:36,531]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 18:09:41,261]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 18:09:41,615]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 18:09:41,640]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 18:09:41,671]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 18:09:41,793]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 9.368 seconds (JVM running for 10.884)
[INFO]-[2019-09-06 18:09:41,975]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:31]-15677643181674787944liupeihao
[INFO]-[2019-09-06 18:09:41,976]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:33]-2f46bd25d611268dd1a218743355d6994feb010a
[INFO]-[2019-09-06 18:09:41,992]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 18:09:41,993]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:11:40,443]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 16288 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 18:11:41,082]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 18:11:41,819]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 18:11:42,165]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$21cf8a8c] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 18:11:42,927]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 18:11:43,399]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:11:43,560]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 18:11:47,828]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 18:11:48,149]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 18:11:48,169]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 18:11:48,194]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 18:11:48,277]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.179 seconds (JVM running for 8.85)
[INFO]-[2019-09-06 18:11:48,428]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:32]-15677643181674787944liupeihao
[INFO]-[2019-09-06 18:11:48,429]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:34]-2f46bd25d611268dd1a218743355d6994feb010a
[INFO]-[2019-09-06 18:11:48,467]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 18:11:48,468]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:18:14,639]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 16392 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 18:18:15,290]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 18:18:16,023]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 18:18:16,358]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$9d03444d] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 18:18:17,122]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 18:18:17,598]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:18:17,776]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 18:18:21,726]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 18:18:22,080]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 18:18:22,104]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 18:18:22,131]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 18:18:22,215]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 7.927 seconds (JVM running for 8.653)
[INFO]-[2019-09-06 18:18:22,366]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:32]-15677643181674787944liupeihao
[INFO]-[2019-09-06 18:18:22,367]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:34]-2f46bd25d611268dd1a218743355d6994feb010a
[INFO]-[2019-09-06 18:18:22,411]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 18:18:22,412]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:35:42,755]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:50]-Starting WchatApplicationTests on LPH with PID 4840 (started by LPH in F:\giuhub\wchat)
[INFO]-[2019-09-06 18:35:43,439]-[192.168.30.2:-1]-[]-[org.springframework.boot.SpringApplication:679]-The following profiles are active: dev
[WARN]-[2019-09-06 18:35:44,218]-[192.168.30.2:-1]-[]-[org.mybatis.logging.Logger:44]-No MyBatis mapper was found in '[com.liupeihao.wchat]' package. Please check your configuration.
[INFO]-[2019-09-06 18:35:44,551]-[192.168.30.2:-1]-[]-[org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker:330]-Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration$$EnhancerBySpringCGLIB$$607ebdc3] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
[INFO]-[2019-09-06 18:35:45,303]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.PropertySourcedRequestMappingHandlerMapping:69]-Mapped URL path [/v2/api-docs] onto method [public org.springframework.http.ResponseEntity<springfox.documentation.spring.web.json.Json> springfox.documentation.swagger2.web.Swagger2Controller.getDocumentation(java.lang.String,javax.servlet.http.HttpServletRequest)]
[INFO]-[2019-09-06 18:35:45,785]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:171]-Initializing ExecutorService 'applicationTaskExecutor'
[INFO]-[2019-09-06 18:35:45,967]-[192.168.30.2:-1]-[]-[com.alibaba.druid.spring.boot.autoconfigure.DruidDataSourceAutoConfigure:56]-Init DruidDataSource
[INFO]-[2019-09-06 18:35:50,338]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:991]-{dataSource-1} inited
[INFO]-[2019-09-06 18:35:50,644]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:160]-Context refreshed
[INFO]-[2019-09-06 18:35:50,662]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.plugins.DocumentationPluginsBootstrapper:163]-Found 1 custom documentation plugin(s)
[INFO]-[2019-09-06 18:35:50,686]-[192.168.30.2:-1]-[]-[springfox.documentation.spring.web.scanners.ApiListingReferenceScanner:41]-Scanning for api listing references
[INFO]-[2019-09-06 18:35:50,791]-[192.168.30.2:-1]-[]-[org.springframework.boot.StartupInfoLogger:59]-Started WchatApplicationTests in 8.433 seconds (JVM running for 9.14)
[INFO]-[2019-09-06 18:35:50,947]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:32]-15677643181674787944liupeihao
[INFO]-[2019-09-06 18:35:50,948]-[192.168.30.2:-1]-[]-[com.liupeihao.wchat.WchatApplicationTests:34]-2f46bd25d611268dd1a218743355d6994feb010a
[INFO]-[2019-09-06 18:35:50,987]-[192.168.30.2:-1]-[]-[com.alibaba.druid.pool.DruidDataSource:1928]-{dataSource-1} closed
[INFO]-[2019-09-06 18:35:50,987]-[192.168.30.2:-1]-[]-[org.springframework.scheduling.concurrent.ExecutorConfigurationSupport:208]-Shutting down ExecutorService 'applicationTaskExecutor'
